_data/reference.yml

```diff
@@ -1,41 +1,62 @@
 bigheader: "Reference Documentation"
 abstract: "Design docs, concept definitions, and references for APIs and CLIs."
 toc:
-- docs/reference.md
+- docs/reference/index.md
 
-- title: "Kubernetes Resource Types"
+- title: "API Reference"
   section:
   - title: Version 1.6
-    path: /docs/resources-reference/v1.6/
+    section:
+    - title: API
+      path: /docs/api-reference/v1.6/
+    - title: Resource Types
+      path: /docs/resources-reference/v1.6/
   - title: Version 1.5
-    path: /docs/resources-reference/v1.5/
+    section:
+    - title: API
+      path: /docs/api-reference/v1.5/
+    - title: Resource Types
+      path: /docs/resources-reference/v1.5/
+  - title: Federation
```

Comment:
this section doesn't fit here. There's a single `Federation` title under `API` section, where are the others (apps, extensions, etc)? This is confusing. Maybe move it under the top level and call it `Federation API`.